src/main/java/sgtravel/storage/Storage.java

```diff
@@ -71,15 +72,15 @@ public Storage() {
         itineraryTable = sampleData.getItineraryTable();
         try {
             read();
-        } catch (FileLoadFailException | ParseException e) {
+        } catch (FileLoadFailException | ParseException | FileNotSavedException e) {
```

Comment:
Why got writing in reading?

Reply:
Cause if it throw error, it would not be able to read the rest of the file. So i call write to write and create a empty file instead.